tests/bud.bats

```diff
@@ -1830,6 +1830,20 @@ _EOF
   test -s "${root}"/subdir/file2.txt
 }
 
+# regression test for https://github.com/containers/podman/issues/10671
+@test "bud-copy-workdir --layers" {
+  _prefetch alpine
+
```

Comment:
@edsantiago has convinced us to create the Containerfiles like we do at line 1810 in this file.  Could you do that rather than creating a separate file?
```
mytmpdir=${TESTDIR}/my-dir
cat > $mytmpdir/Containerfile.2 << _EOF
FROM alpine
WORKDIR /subdir
COPY file1.txt /subdir
RUN ls
_EOF
```

and then adjusting your bud command below

Reply:
@TomSweeneyRedHat this is embarrassing, when did I make that recommendation? The case in 1810 is, I think, due to the need for an ignore file in the same directory. The overwhelming majority of buildah tests use a separate Docker/Containerfile, and I think I'm fine with that?